<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HazardZonesByFlowDockWidgetBase</class>
 <widget class="QDockWidget" name="HazardZonesByFlowDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hazard Zones by Flow</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="1">
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string>%p%</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="startPushButton">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Start Process</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="statusLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>104</width>
        <height>0</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Changing pixel values</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="rasterGroupBox">
      <property name="title">
       <string>Raster Band</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QComboBox" name="rasterComboBox"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="sourceRasterPathLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Directory</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Layer</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QGroupBox" name="unitsGroupBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Units</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="siRadioButton">
            <property name="text">
             <string>International System</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="usRadioButton">
            <property name="text">
             <string>US customary </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="HydraulicGroupBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Type of hydraulic data</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="caladoRadioButton">
            <property name="text">
             <string>Depth</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="velocidadRadioButton">
            <property name="text">
             <string>Velocity</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="productRadioButton">
            <property name="text">
             <string>Depth per Velocity</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Result layer</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="1">
        <widget class="QLineEdit" name="OuptDir">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The directory where the output is stored.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The directory where the output is stored.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string>Format</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QCheckBox" name="resultCheckBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Add result to project</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="cmdBrowse">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The directory where the output is stored.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string>Name layer (optional)</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="formatComboBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="nameoutLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="inputMask">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string notr="true">e.g. output_raster</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The directory where the output is stored.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string>Output layer</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>label_8</zorder>
      <zorder>formatComboBox</zorder>
      <zorder>label_9</zorder>
      <zorder>cmdBrowse</zorder>
      <zorder>label_7</zorder>
      <zorder>OuptDir</zorder>
      <zorder>nameoutLineEdit</zorder>
      <zorder>resultCheckBox</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
